src/index.ts

```diff
@@ -1899,9 +1899,10 @@ function FlatpickrInstance(
       positionCalendar(positionElement);
     }
 
-    if (self.config.enableTime === true && self.config.noCalendar === true) {
-      if (
-        self.config.allowInput === false &&
+    if(self.config.noCalendar === false) {
```

Comment:
I'm not that familiar with the flatpickr codebase yet, so this may be too broad of a check.  Maybe also include a `self.config.wrap === true` check?